snuba/datasets/processors/replays_processor.py

```diff
@@ -196,6 +200,9 @@ def process_message(
             self._process_event_hash(processed, replay_event)
             self._process_contexts(processed, replay_event)
             return InsertBatch([processed], None)
+        except LoggedException:
+            logger.error("A replay schema constraint was not observed.", exc_info=True)
```

Comment:
Do you mind changing this to logger.warning instead? Sentry alerts for SnS counts number of errors and I believe these would still trigger our alerts.

Reply:
@nikhars 👍